### ISSUE TITLE
Standardize formatting in Suggested Lesson Tables

### ISSUE THREAD

**Issue by ndporter** [carpentries/instructor-training#1428]
Clean up minor inconsistencies in [suggested lessons](https://carpentries.github.io/instructor-training/demo_lessons/index.html) including:
* consistent phrasing, capitalization, and punctuation for explanations (e.g. "none" vs. "No episodes"; "Avoid all episodes.", etc)
* consistent language use in the Spanish lessons (one note in English, rest in Spanish)
* Consistent decision to use unordered lists (`*`) or line breaks (`<br/>`) to separate rows in good/avoid lessons cells (ex: SC R Gapminder uses rows while most use lists)

Additionally, I suggest we consider using lists (`*`) in all good/avoid lesson cells, even when it's just one item for consistency.

**Comment by brownsarahm** [comment]
I like these ideas. We had a previous PR to make some progress on the consistency point, which mades some progress, but may have missed some (#1241 ) 

I think we should:
- finish dealing with #1407 and get that merged before making big updates.
- find out if the workbench supports easier to read tables eg maybe something like the [myst markdown list table](https://jupyterbook.org/en/stable/reference/cheatsheet.html#tables). If it does, we can make changes after when it's an easier to read page.

**Comment by ndporter** [comment]
Agreed on all points. I made this issue rather than a PR specifically to avoid creating conflicts with current PRs and issues. I apologize for missing #1241; I did look for open issues but that slipped by.
